### ISSUE TITLE
Add option unsleep with play music

### ISSUE THREAD

**Issue by borut1** [adlerluiz/ytmdesktop#1404]
### Consent

- [X] I verified that there is no open/closed issue for the same subject.
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Feature

The option for the computer to wake up the computer from computer sleep (continuous computer mode) while playing YTD applications is missing. Currently, you have to change the power plan or use other software that wakes up the computer for the period of required work.

### Solution

Add blocking computer sleep with play music/work YTD

**Comment by Alipoodle** [comment]
This sounds more like you need to adjust your Battery settings to your preference of not going to sleep while Audio is playing.
YTMD Correctly flags to Windows that Audio/Video is playing correctly.

_YTMD is Playing Audio, and FireFox is playing a video in this_
![image](https://github.com/user-attachments/assets/ceed3074-4edd-4124-82e5-22138fcb616b)

**Comment by borut1** [comment]
Yes, it would be useful to have an option to put the computer in Unsleep (off sleep mode (display, system, etc), like Sleep Timer on the bottom bar of the program.

**Comment by borut1** [comment]
Yes, it would be useful to have an option to put the computer into awaked mode (disabled sleep mode display, system, etc.), using the example of the timer on the bottom bar of the program.

**Comment by zenjr** [comment]
Hi @Alipoodle , indeed windows 11 does report that there an audio source playing, windows itself does nothing with that information. There used to be an option to prevent it to sleep when audio was playing but 11 does not have that anymore.  

Try issuing a `powercfg /query` and you will see that there is nothing there to set to avoid sleep when audio is playing , the legacy GUI also doesn't have anything: ![Image](https://github.com/user-attachments/assets/b5c92aa6-5914-47b3-9fd8-0f0c4de9c98e)
